docs/application/native/guides/text-input/autofill.md

```diff
@@ -308,7 +292,7 @@ To retrieve the data previously entered by the user and stored in an Autofill se
     autofill_view_info_destroy(vi_h);
     ```
 
-## Storing Entered Data
+## Save Autofill Data
 
```

Comment:
Line 299:  Create an Autofill data for each input fields, to save:
Line 313 : Create an Autofill view data, to include multiple Autofill items:
Line 332: Request with the `autofill_commit()` function, to store the data in an Autofill service using `autofill_commit()`:

Reply:
Fixed